### PR TITLE
feat(testrunner): cache transformed files

### PR DESCRIPTION
**Pull request by JoelEinbinder**
This makes parsing faster to the point where it is not worth it to report when files are parsed.

### CODE REVIEW

File: test/runner/transform.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const version = 0;
+const cacheDir = path.join(os.tmpdir(), 'playwright-transform-cache');
+
+/**
+ * @param {string} content 
+ * @param {string} filePath 
+ * @return {string}
+ */
+function calculateCachePath(content, filePath) {
+  const hash = crypto.createHash('md5').update(content).update(filePath).update(String(version)).digest('hex');
```
Comment (pavelfeldman) [inline test/runner/transform.js:16]: sha1

File: test/runner/transform.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const fileName = path.basename(filePath, path.extname(filePath)).replace(/\W/g, '') + '_' + hash;
```
Comment (pavelfeldman) [inline test/runner/transform.js:17]: just use hash?

**Review by pavelfeldman — APPROVED** [review]

File: test/runner/transform.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const crypto = require('crypto');
```
[the remark this answers is not in the document]
Comment (JoelEinbinder) [reply test/runner/transform.js:16]: done

File: test/runner/transform.js (hunk 2 above)
Comment (JoelEinbinder) [reply test/runner/transform.js:32]: Copied this from jest. putting the name in the cache path makes it way easier to debug issues with the cache system.

**Pull request merged**